doc/classes/EditorSettings.xml

```diff
@@ -861,7 +869,7 @@
 			[b]Note:[/b] Defaults to [code]1[/code] on non-touchscreen devices.
 		</member>
 		<member name="network/connection/network_mode" type="int" setter="" getter="">
-			Determines whether online features are enabled in the editor, such as the Asset Library. Setting this property to "Offline" is recommended to limit editor's internet activity, especially if privacy is a concern.
+			Determines whether online features are enabled in the editor, such as the Asset Library or update checks. Setting this property to "Offline" is recommended to limit editor's internet activity, especially if privacy is a concern.
```

Comment:
I know it's pre-existing, but I don't think we should say setting this to "Offline" is _recommended_.
```suggestion
			Determines whether online features are enabled in the editor, such as the Asset Library or update checks. Disabling these online features helps alleviate privacy concerns by preventing the editor from making HTTP requests to the Godot website, GitHub, or third-party platforms hosting assets from the Asset Library.
```